sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -3582,6 +3600,35 @@ public void run() {
 		SdlTrace.logProxyEvent("Proxy received RPC Message: " + functionName, SDL_LIB_TRACE_KEY);
 	}
 
+	//FIXME
+	/**
+	 * Temporary method to bridge the new PLAY_PAUSE and OKAY button funcitonliaty with the old
+	 * OK button name. This should be removed during the next major release
+	 * @param notification
+	 */
+	private RPCNotification handleButtonNotificationFormatting(RPCNotification notification){

```

Comment:
It looks like you don't need to create new `RPCNotification` objects here:

```
	private RPCNotification handleButtonNotificationFormatting(RPCNotification notification){
		if(FunctionID.ON_BUTTON_EVENT.toString().equals(notification.getFunctionName())
				|| FunctionID.ON_BUTTON_PRESS.toString().equals(notification.getFunctionName())){

			ButtonName buttonName = (ButtonName)notification.getObject(ButtonName.class, OnButtonEvent.KEY_BUTTON_NAME);
			if(rpcSpecVersion != null && rpcSpecVersion.getMajor() >= 5){
				if(ButtonName.PLAY_PAUSE.equals(buttonName)){
					notification.setParameters(OnButtonEvent.KEY_BUTTON_NAME, ButtonName.OK);
					return notification;
				}
			}else{
				if(ButtonName.OK.equals(buttonName)){
					notification.setParameters(OnButtonEvent.KEY_BUTTON_NAME, ButtonName.PLAY_PAUSE);
					return notification;
				}
			}

		}
		return null;
	}
```

Will fix the CCE errors above